test/src/specs/tx_pool/different_txs_with_same_input.rs

```diff
@@ -19,15 +20,10 @@ impl Spec for DifferentTxsWithSameInput {
         let tx1 = node0.new_transaction(tx_hash_0.clone());
         let tx2_temp = node0.new_transaction(tx_hash_0.clone());
         // Set tx2 fee to a higher value
-        let output = tx2_temp
-            .outputs()
-            .as_reader()
-            .get(0)
-            .unwrap()
-            .to_entity()
-            .as_builder()
-            .capacity(capacity_bytes!(40_000).pack())
+        let output = CellOutputBuilder::default()
+            .capacity(capacity_bytes!(100).pack())
```

Comment:
Before the output capacity was too high, the transaction returned error while RPC sending.